app/models/volunteer.rb

```diff
@@ -2,6 +2,15 @@
 class Volunteer < ActiveRecord::Base
   default_scope { order('volunteers.name ASC').where(active: true) }
 
+  scope :in_regions, ->(regions) { joins(:regions).where('regions.id IN (?)', Array.wrap(regions)).uniq }
+
+  scope :needing_training, lambda {
+    joins('LEFT JOIN log_volunteers ON log_volunteers.volunteer_id = volunteers.id')
+    .joins('LEFT JOIN logs ON logs.id = log_volunteers.log_id AND logs.complete = TRUE')
```

Comment:
This will work as long as volunteers cannot have both complete and incomplete logs.  If that's possible, this will require a limited subquery instead

Reply:
Could you explain why you're checking for `complete: true` here? Does it do some kind of 'or' logic to get the incomplete logs? 

Thanks for the PR and sorry again about the slow response!

Reply:
Ahhh the `Left Join` does it!